benefits/enrollment/views.py

```diff
@@ -93,7 +94,8 @@ def _enroll(request):
         raise Exception("Updated customer_id does not match enrolled customer_id")
 
 
-@decorator_from_middleware(middleware.EligibleSessionRequired)
+@decorator_from_middleware(EligibleSessionRequired)
+@decorator_from_middleware(LoginRequired)
```

Comment:
Now that we're seeing more about how the sign out works in #436, I think we don't want this middleware on the `enrollment` views at all.

For one, `EligibleSessionRequired` already blocks users that have not successfully gone through Eligibility Verification, and doing so means they would have already had to login.

And now that we're signing users out and needing to bring them back into the app at different points (including 2 in `enrollment`) - `LoginRequired` here would stop that from working.